business_requirement_deliverable_sale/wizard/convert_requirement_sale.py

```diff
@@ -0,0 +1,135 @@
+# Copyright 2019 Tecnativa Victor M.M. Torres>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, models, fields, _
+
+
+class ConvertRequirementSale(models.TransientModel):
+    """Wizard to guide create new SO from requirement"""
+
+    _name = 'convert.requirement.sale'
```

Comment:
Call it with the same prefix as the rest: `business.requirement.convert.sale` or similar.